pom.xml

```diff
@@ -126,7 +126,7 @@
         <version.org.apache.james.apache-mime4j>0.6</version.org.apache.james.apache-mime4j>
         <version.org.apache.lucene>5.3.1</version.org.apache.lucene>
         <version.org.apache.myfaces.core>2.2.11</version.org.apache.myfaces.core>
-        <version.org.apache.openjpa>2.4.1</version.org.apache.openjpa>
+        <version.org.apache.openjpa>2.4.2</version.org.apache.openjpa>
```

Comment:
This is **testsuite only** dependency, upgraded as it includes fix to work better on WildFly see 
https://issues.apache.org/jira/browse/OPENJPA-2684 - Persistence entities not recognized in Wildfly 10 if in a JAR
full release notes: 
http://openjpa.apache.org/openjpa-2.4.x.html#changes-in-openjpa-242 for full change list.